api/graphql/schema.graphql

```diff
@@ -478,6 +483,15 @@ type PoolCandidate {
   signature: String
   submittedSteps: [ApplicationStep!] @rename(attribute: "submitted_steps")
   screeningQuestionResponses: [ScreeningQuestionResponse] @hasMany
+  minimumCriteria: ApplicationCriteria @rename(attribute: "minimum_criteria")
+
+  # experiences for meeting minimum criteria
+  awardExperiencesCriteria: [AwardExperience] @belongsToMany
+  communityExperiencesCriteria: [CommunityExperience] @belongsToMany
+  educationExperiencesCriteria: [EducationExperience] @belongsToMany
+  personalExperiencesCriteria: [PersonalExperience] @belongsToMany
+  workExperiencesCriteria: [WorkExperience] @belongsToMany
```

Comment:
I don't think we'll ever want these criteria experiences divided by type, so we can simplify by removing these, and just leaving the general `[Experience]` field.

Reply:
Done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/6379/commits/fd874cd0c1a0aa13b36f92256b11a2fad0ea68ba